pkg/scheduler/scheduling_application.go

```diff
@@ -409,6 +409,21 @@ func (sa *SchedulingApplication) sortRequests(ascending bool) {
 	}
 }
 
+func (sa *SchedulingApplication) getOutstandingRequests(headRoom *resources.Resource, total *outstandingRequests) {
```

Comment:
Suggest to rename to `getAllocatableOutstandingRequests`

Reply:
The requests are not allocatable, if they were we would and should not see them in the list.

Reply:
Since there is no consensus on the name of this function, I'd prefer to keep it as it is for now.
We can revisit this if we can come up with a better name, this is a non-functional change, not critical.